sphinx/ext/inheritance_diagram.py

```diff
@@ -57,12 +57,132 @@ class E(B): pass
 from sphinx.util import force_decode
 from sphinx.util.compat import Directive
 
+import collections
+import collections.abc
```

Comment:
This module does not exist in 2.x, but Iterable is always available directly in `collections`.